src/tools/user-agent-explainer/user-agent-explainer.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div>
+    <n-space justify="center">
+      <n-form-item label="Use user agent from local browser?" label-placement="left" :show-feedback="false">
+        <n-switch v-model:value="useCurrentUserAgent" @click="useCurrentUserAgentClick" />
+      </n-form-item>
+    </n-space>
```

Comment:
I think it's useless to have a switch to use the current user agent, we can just put the current user UA as the default value.
```suggestion
```

Reply:
See next comment.